eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/eventlist/view/FlexibleAdapterEventListView.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2016 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.eventlist.view;
+
+import android.support.v7.widget.RecyclerView;
+
+import com.schedjoules.eventdiscovery.eventlist.itemsprovider.EventListItemsProviderImpl;
+
+import eu.davidea.flexibleadapter.FlexibleAdapter;
+import eu.davidea.flexibleadapter.items.IFlexible;
+
+
+/**
+ * Handles the RecyclerView and the Adapter initialization for the Event list.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class FlexibleAdapterEventListView implements EventListView
```

Comment:
What's the real "single purpose" if this class? The name says it's a "View". The description says it "handles" something. To me it looks like an alibi class for something that was meant to go into a static method.
This should be refactored. Do we really need this "class"?